.github/workflows/update_tf.yml

```
@@ -48,9 +50,11 @@ jobs:
           body: |
             Updates TF to
             [${{ env.TF_SHA }}](https://github.com/tensorflow/tensorflow/commit/${{ env.TF_SHA }})
+            matching
+            [${{ env.TF_SHA }}](https://github.com/llvm/llvm-project/commit/${{ env.TF_SHA }})
```

Should this be env.LLVM_SHA in both instances on this line? Or TF_SHA for both tensorflow and llvm...?

```suggestion
            [${{ env.LLVM_SHA }}](https://github.com/llvm/llvm-project/commit/${{ env.LLVM_SHA }})
```

---

Huh I definitely changed that... Maybe I was in the wrong branch. Thanks for catching :-)